CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ endif()
 project(HPX CXX C)
 
 set(HPX_MAJOR_VERSION 1)
-set(HPX_MINOR_VERSION 1)
+set(HPX_MINOR_VERSION 2)
 set(HPX_PATCH_LEVEL   0)
 set(HPX_VERSION "${HPX_MAJOR_VERSION}.${HPX_MINOR_VERSION}.${HPX_PATCH_LEVEL}")
 set(HPX_LIBRARY_VERSION "${HPX_VERSION}")
```

Comment:
In the future we might want to generate our version.hpp content from these definitions, that should simplify things a bit...